wordpress-mobile/WordPress-iOS
# Save for later: toggle button in article details

Implements #9266 

**This PR does not have a milestone yet**.

This PR starts where the [previous one](https://github.com/wordpress-mobile/WordPress-iOS/pull/9279) ends, adding:

- Toggle button in article details 

![simulator screen shot - iphone 8 - 2018-05-07 at 15 55 50](https://user-images.githubusercontent.com/2722505/39691459-34d95672-5210-11e8-8398-cd5697df1c50.png)
![simulator screen shot - iphone 8 - 2018-05-07 at 15 55 53](https://user-images.githubusercontent.com/2722505/39691470-39eb9828-5210-11e8-8d6e-2ba10e61958c.png)
![simulator screen shot - iphone 8 - 2018-05-07 at 15 56 01](https://user-images.githubusercontent.com/2722505/39691476-3f989c76-5210-11e8-85d4-6d9f10bc58b3.png)


To test:
1. Deactivate the feature flag
2. Navigate to reader.
3. Tap a post, to navigate to the details screen
4. The bookmark button should not be visible
5. Activate the feature flag
6. Navigate to the same screen. The bookmark button should be visible.
Tapping the bookmark button should make it change color (to blue). The ReaderPostService should log a message in the console (to be removed when it is actually implemented). A Notice should be presented.
